gensim/corpora/wikicorpus.py

```diff
@@ -544,10 +555,15 @@ def __init__(self, fname, processes=None, lemmatize=utils.has_pattern(), diction
             Maximal token length.
         lower : bool, optional
              If True - convert all text to lower case.
+        filter_articles: callable or None, optional
+            If set, each XML article element will be passed to this callable before being processed. Only articles
+            where the callable returns an XML element are processed, returning None allows filtering out
+            some articles based on customised rules.
```

Comment:
Can you create an example of this function and link it in docstring like
```
:func:`gensim.corpora.wikicorpus.filter_example`
```
this can be useful for better understanding, how function works
- input parameters
- return values (probably True/False better that XML element/None)

Reply:
you mean add an example `filter_example` to the `wikicorpus` module?

Reply:
got it, very busy at work but will try to get it done this week

Reply:
don't forget to resolve merge conflict too